librepo/CMakeLists.txt

```diff
@@ -60,9 +67,12 @@ TARGET_LINK_LIBRARIES(librepo
                         ${LIBXML2_LIBRARIES}
                         ${CURL_LIBRARY}
                         ${LIBCRYPTO_LIBRARIES}
-                        ${GPGME_VANILLA_LIBRARIES}
                         ${GLIB2_LIBRARIES}
+                        ${RPM_LIBRARIES}
```

Comment:
Is there a reason why this is not symmetric as with conditionally adding the `${GPGME_VANILLA_LIBRARIES}` below?

Reply:
No. Forgotten from testing.